omas/machine_mappings/d3d.json

```diff
@@ -113,32 +113,39 @@
  "coils_non_axisymmetric.coil.:.name": {
   "PYTHON": "coils_non_axisymmetric_hardware(ods, {pulse})"
  },
- "core_profiles.global_quantities.v_loop": {
-  "COCOSIO": 11,
-  "PYTHON": "core_profiles_global_quantities_data(ods, {pulse})"
- },
```

Comment:
I had to rip this out to make it work. Putting it back in causes:
``core_profiles.profiles_1d.0.grid.rho_pol_norm` has no data`

Reply:
@orso82 Do you understand why @AreWeDreaming would get this error, if this is not removed?

Reply:
Maybe `core_profiles_global_quantities_data ` looks at `core_profiles.profiles_1d.0.grid.rho_pol_norm` for some reason, and that mapping was not in place?